config.json

```diff
@@ -115,6 +115,15 @@
       ]
     },
     {
+      "uuid": "40d4bb98-2e4a-4579-810e-g1230bc158e9",
```

Comment:
note: check whether g is OK for uuid

Reply:
AFAIK UUID is usually 16 byte written in hexadecimal notation and separated into 5 groups. So it doesn't look like a valid UUID to me.

Reply:
Has this UUID been generated by configlet?

Reply:
I've opened a bug in the configlet repo.

Wether generated or not, at least the linter should have complained:

exercism/configlet#100

Reply:
@NobbZ It wasn't generated by configlet, my bad there. Also you might want to specify in the issue that it has already been fixed the UUID of this repo.